src/EventListener/ParamConverterListener.php

```diff
@@ -106,19 +106,12 @@ private function autoConfigure(\ReflectionFunctionAbstract $r, Request $request,
 
     private function getParamClassByType(?\ReflectionType $type): ?string
     {
-        if (\PHP_VERSION_ID < 80000) {
```

Comment:
💡 We don't need no special treatment for PHP 7 here. The condition `$type instanceof \ReflectionUnionType` below already does the trick by always evaluating to `false` on PHP 7.